infra/src/database.ts

```diff
@@ -46,6 +57,205 @@ const dbInstance = new aws.rds.Instance(nm("app"), {
   finalSnapshotIdentifier: nm("app-db-final-snapshot"),
 });
 
+const encryptedDbInstance = new aws.rds.Instance("app-encrypted", {
+  username: "openpipe",
+  password: password.result,
+  dbName,
+  instanceClass: "db.m7g.2xlarge",
+  allocatedStorage: 1210,
+  maxAllocatedStorage: 2000,
+  storageType: "io1",
+  iops: 12000,
+  engine: "postgres",
+  engineVersion: "15.5",
+  parameterGroupName: "default.postgres15",
+  dbSubnetGroupName: dbSubnetGroup.name,
+  vpcSecurityGroupIds: [dbSecurityGroup.id],
+  publiclyAccessible: true,
+  applyImmediately: true,
+  backupRetentionPeriod: 30,
+  performanceInsightsEnabled: true,
+  multiAz: isProd,
+  skipFinalSnapshot: !isProd,
+  // snapshotIdentifier: isProd ? undefined : "app-pl-stage-test-snapshot",
+  finalSnapshotIdentifier: nm("app-db-encrypted-final-snapshot"),
+  storageEncrypted: true,
+  identifier: "app-encrypted-db-instance",
```

Comment:
We should just let pulumi set the name. Hardcoding it like this will fail in prod anyway, since it'll try to use the same name for the existing staging one and fail because the name has to be unique.